src/ControlPanel.jsx

```diff
@@ -81,8 +81,8 @@ class ControlPanel extends Component {
         <div className="routes-header stops-toggle">
           <h3>Stops</h3>
           <Toggle
-            defaultChecked={this.state.showStops}
-            onChange={() => this.setState({ showStops: !this.state.showStops })}
+            defaultChecked={1}
```

Comment:
would this work if we kept it with `this.state.showStops` (which is set to true)

Reply:
no, `this.state.showStops` isn't stored in the control panel component, it is in the parent Map component

Reply:
```suggestion
             defaultChecked
```